### ISSUE TITLE
Screenreader issue for <aria-labelledby> attribute

### ISSUE THREAD

**Issue by sswweber** [vaadin/vaadin-text-field-flow#299]
The description of text fields cannot be recognized by the screenreader. The <aria-labelledby> attribute is filled with the id values of the label and the respective text field. 
example: `aria-labelledby="vaadin-text-field-label-0 vaadin-text-field-input-0`.
